nbgrader/tests/apps/test_nbgrader_submit.py

```diff
@@ -231,7 +232,7 @@ def test_submit_include(self, exchange, cache, course_dir):
         filename, = os.listdir(join(exchange, "abc101", "inbound"))
         assert not exists(join(exchange, "abc101", "inbound", filename, "foo.txt"))
 
-    def test_submit_include(self, exchange, cache, course_dir):
```

Comment:
This is renamed since it's the same name as above - it was probably named wrong.